package.json

```diff
@@ -439,6 +439,12 @@
                     "default": true,
                     "description": "Clear the output channel at the beginning of a build",
                     "scope": "resource"
+                },
+                "makefile.compileCommandsPath": {
+                    "type": "string",
+                    "default": ".vscode/compile_commands.json",
```

Comment:
It would seem that we are always writing this file as a result. We may need a separate setting to enable/disable it.  Another option is to default this to empty string and not do the compile commands unless it is explicitly set.  This could also be enforced with the `inspect` API.  The advantage of using that API is that vscode will autofill the default when you override it (I think?)

Reply:
My mistake. Setting it to null then, to match the extensionLog setting. No writing it unless the user specifies a path.